Makefile

```diff
@@ -1,15 +1,140 @@
+CNone  := $(shell printf "\033[0m")
+
+# ====================================================================================
+# Logger
+
+TIME_LONG	= `date +%Y-%m-%d' '%H:%M:%S`
+TIME_SHORT	= `date +%H:%M:%S`
+TIME		= $(TIME_SHORT)
+
+INFO = echo ${TIME} ${BLUE}[ .. ]${CNone}
+WARN = echo ${TIME} ${YELLOW}[WARN]${CNone}
+ERR  = echo ${TIME} ${RED}[FAIL]${CNone}
+OK   = echo ${TIME} ${GREEN}[ OK ]${CNone}
+FAIL = (echo ${TIME} ${RED}[FAIL]${CNone} && false)
+
+# ====================================================================================
+# Verbosity control hack
+
+VERBOSE ?= 0
+AT_0 := @
+AT_1 :=
+AT = $(AT_$(VERBOSE))
+
+# ====================================================================================
+# Targets
+
+help: ## to get help
+	$(AT)echo "Welcome to ${APP_NAME}:${APP_VERSION}"
+	$(AT)echo "Usage:"
+	$(AT)grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) |\
+	awk 'BEGIN {FS = ":.*?## "}; {printf "make ${CYAN}%-30s${CNone} %s\n", $$1, $$2}'
+
+.PHONY: python-build
+python-build: ## to build python package
+	$(AT)$(INFO) python build $*...
+	$(AT)$(POETRY) build || ${FAIL}
+	$(AT)$(OK) python build $*
+
+.PHONY: python-test
+python-test: ## to run python tests
+	$(AT)$(INFO) testing python...
+	$(AT)$(POETRY) run pytest || ${FAIL}
+	$(AT)$(OK) testing python
+
+.PHONY: python-update-dependencies
+python-update-dependencies: ## to update python dependencies
+	$(AT)$(INFO) updating python dependencies...
+	$(AT)$(POETRY) install && \
+	    $(POETRY) run pip install clearbit==0.1.7 || ${FAIL}
+	$(AT)$(OK) updating python dependencies
+
+.PHONY: dbt-docs
+dbt-docs: ## to generate and serve dbt docs
+	$(AT)$(INFO) Generating docs and spinning up the a webserver on port 8081...
+	$(AT)export DBT_PROFILE_PATH=".";\
+	$(DOCKER) compose run -d -p "8081:8081" dbt_image bash -c "dbt deps && dbt docs generate -t prod && dbt docs serve --port 8081"  || ${FAIL}
+	$(AT)$(OK) Server started, visit http://localhost:8081 to view the docs
+
+.PHONY: dbt-generate-docs
+dbt-generate-docs: ## to generate dbt docs
+	$(AT)$(INFO) Generating docs...
+	$(AT)export DBT_PROFILE_PATH=".";\
+	$(DOCKER) compose run dbt_image bash -c "dbt deps && dbt docs generate -t prod" } || ${FAIL}
+	$(AT)$(OK) Generated docs
+
+.PHONY: dbt-bash
+dbt-bash: ## to start a bash shell with DBT
+	$(AT)$(INFO) Running bash with dbt...
+	$(AT)export DBT_PROFILE_PATH=".";\
+	$(DOCKER) compose -f ${DOCKER_COMPOSE_DBT_FILE} run dbt_image bash -c "dbt deps && /bin/bash" || ${FAIL}
+	$(AT)$(OK) Exited dbt bash
+
+.PHONY: data-image
+data-image: ## to start a bash shell on the data image
+	$(AT)$(INFO) Attaching to data-image and mounting repo...
+	$(AT)export DBT_PROFILE_PATH=".";\
+	$(DOCKER) compose -f ${DOCKER_COMPOSE_DBT_FILE} run data_image bash || ${FAIL}
+	$(AT)$(OK) Exited data-image
+
+.PHONY: clean
+clean: ## to clean-up
+	@$(INFO) cleaning ${PYTHON_OUT_BIN_DIR} folder...
+	$(AT)rm -rf ${PYTHON_OUT_BIN_DIR} || ${FAIL}
+	@$(OK) cleaning ${PYTHON_OUT_BIN_DIR} folder
```

Comment:
Any reason for not including the verbosity flag here ? `$(AT)` ?

Reply:
Most likely copy-paste error, fixed :)

Reply:
@ifoukarakis Thanks! 

Also there are some tests failing